copier_template_tester/main.py

```diff
@@ -63,11 +82,14 @@ def run(base_dir: Path | None = None) -> None:
     defaults = config.get('defaults', {})
 
     input_path = base_dir
+    output_paths = set()
     for key, data in config['output'].items():
         output_path = base_dir / key
         output_path.mkdir(parents=True, exist_ok=True)
+        output_path.add(output_path)
```

Comment:
I tested it and it failed with:

```

copier-template-tester...................................................Failed
- hook id: copier-template-tester
- exit code: 1

Warning: You probably want a section: [defaults]
Traceback (most recent call last):
  File "/nix/store/3dq3n73wf7zc5j51bc7sr3939mbvdxp4-python3.10-copier-template-tester-1.0.2/bin/.ctt-wrapped", line 9, in <module>
    sys.exit(run())
  File "<@beartype(copier_template_tester.main.run) at 0x7fd9ecd50280>", line 36, in run
  File "/nix/store/a0dqd5bia22xinhwxp241c59376xf5ld-python3-3.10.9-env/lib/python3.10/site-packages/copier_template_tester/main.py", line 89, in run
    output_path.add(output_path)
AttributeError: 'PosixPath' object has no attribute 'add'
```

Reply:
Thanks for bumping this issue and trying out the changes!